README.md

```diff
@@ -1,216 +1,265 @@
-}
-```
+1. In Android Studio, run the `iosApp` configuration with the selected real device. The build should fail.
+2. Go to Xcode and select **Open a project or file**.
+3. Navigate to the `iosApp/iosApp.xcworkspace` file of your project.
+4. In the left menu, select `iosApp`.
+5. Navigate to **Signing & Capabilities**.
+6. In the **Team** list, select your personal team.
 
-Update the shared code by adding a text field that will update the name displayed on the button:
-
-```diff
-@OptIn(ExperimentalResourceApi::class)
-@Composable
-internal fun App() {
-    MaterialTheme {
-        var greetingText by remember { mutableStateOf("Hello, World!") }
-        var showImage by remember { mutableStateOf(false) }
-        Column(Modifier.fillMaxWidth(), horizontalAlignment = Alignment.CenterHorizontally) {
-            Button(onClick = {
-                greetingText = "Hello, ${getPlatformName()}"
-                showImage = !showImage
-            }) {
-                Text(greetingText)
-            }
-+           TextField(greetingText, onValueChange = { greetingText = it })
-            AnimatedVisibility(showImage) {
-                Image(
-                    painterResource("compose-multiplatform.xml"),
-                    null
-                )
-            }
-        }
-    }
-}
-```
+   If you haven't set up your team yet, use the **Add account** option and follow the steps.
 
-<img src="readme_images/text_field_added.png" height="200px">
-
-### Configuring the iOS application
-
-This template contains a `iosApp/Configuration/Config.xcconfig` configuration file that allows you to configure most basic properties without having to move to Xcode. It contains:
-- `APP_NAME` - target executable and application bundle name
-- `BUNDLE_ID` - [bundle identifier](https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleidentifier#discussion)
-- `TEAM_ID` - [Team ID](https://developer.apple.com/help/account/manage-your-team/locate-your-team-id/#:~:text=A%20Team%20ID%20is%20a,developer%20in%20App%20Store%20Connect.)
-
-Note: To configure the `APP_NAME` setting, open `Config.xcconfig` in any text editor *before opening* the project in Android Studio, and set the desired name.
-
-If you need to change this setting after you open the project in Android Studio, please do the following:
-- close the project in Android Studio
-- run `./cleanup.sh` in your terminal
-- change the setting
-- open the project in Android Studio again
+</details>
 
-For configuring advanced settings, you can use Xcode. Open the `iosApp/iosApp.xcworkspace` in Xcode after opening the project in Android Studio, and use Xcode to make your changes.
+To run the application, set the `TEAM_ID`:
+
+1. In the template, navigate to the `iosApp/Configuration/Config.xcconfig` file.
+2. Set your `TEAM_ID`.
+3. Re-open the project in Android Studio. It should show the registered iOS device in the `iosApp` run configuration.
+
+## Make your first changes
+
```

Comment:
I would add an intro here on why we recommend that you do the first changes. Something like to feel benefits of cross-platform development of UIs.

Reply:
Sure, something along the lines of 'you can modify the code, and your changes will be visible on both iOS and Android' / 'let's make some changes and see how they work on ios and android out of the box'

Reply:
Good idea! Added a new paragraph

Reply:
LGTM!

Reply:
LGTM!